desktop/src/main/java/bisq/desktop/main/dao/governance/result/VoteResultView.java

```diff
@@ -358,9 +363,27 @@ private void fillCycleList() {
         });
         Collections.reverse(cycleListItemList);
 
+        maybeShowDAOTestingFeedbackWindow();
+
         GUIUtil.setFitToRowsForTableView(cyclesTableView, 25, 28, 2, 4);
     }
 
+    private void maybeShowDAOTestingFeedbackWindow() {
+        String testingPopupKey = "daoTestingFeedbackPopup";
+        if (DontShowAgainLookup.showAgain(testingPopupKey)) {
+            UserThread.runAfter(() -> {
+                if (sortedCycleListItemList.size() > 0 && (sortedCycleListItemList.get(0).getResultsOfCycle().getEvaluatedProposals().stream().map(EvaluatedProposal::getProposalTxId)
+                        .flatMap(key -> bsqWalletService.getWalletTransactions().stream().map(Transaction::getHashAsString)
```

Comment:
I would prefer here also to use the DAO domain instead the BTC domain to avoid too much dependency to the wallet.
MyBlindVoteListService holds your own blind votes and with a check for the cycle you see if it is a current vote.